assets/src/edit-story/app/media/utils/createResource.js

```diff
@@ -85,8 +86,7 @@ import getTypeFromMime from './getTypeFromMime';
  * @property {boolean} local Whether the resource has been already uploaded to
  * the server.
  * @property {Object} sizes Object of image sizes.
- * @property {?Attribution} attribution An optional attribution for the
- * resource.
+ * @property {Attribution} attribution An attribution for the resource.
```

Comment:
This should probably be optional, as Coverr doesn't have per-resource attribution I believe

Reply:
Done.